src/navigation/FooterNav.scss

```diff
@@ -29,9 +32,11 @@
   }
 
   .footer-copyright {
+    color: var(--copyright-text-color);
     width: auto;
     flex-shrink: 0;
     @screen lg {
+      @apply w-full;
```

Comment:
@emilyjablonski looks like this change is changing the layout of the links on the footer. They are now stacked on top of each other. 
![image](https://user-images.githubusercontent.com/42942267/226941119-87dc3168-de61-424d-a4af-d0a0a8f709be.png)